src/scenic/simulators/carla/simulator.py

```diff
@@ -172,3 +194,18 @@ def getProperties(self, obj, properties):
 			angularSpeed=utils.carlaToScenicAngularSpeed(currAngVel),
 		)
 		return values
+
+	def destroy(self):
+		for obj in self.objects:
```

Comment:
This also looks like it got pulled in from #11; we may need to wait until that is merged before fixing this one (I'm not sure how to do dependencies between PRs on GitHub).

Reply:
Indeed, this PR adds the `client.stop_recorder`, which is why the whole destroy function was brought